<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finishedProc</class>
 <widget class="QWidget" name="finishedProc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>61</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="idLB">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="operatorLB">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Operator:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resultLB">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
